tests/test_instrument.py

```diff
@@ -0,0 +1,61 @@
+# This file is part of obs_decam.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (http://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Tests of the DarkEnergyCamera instrument class.
+"""
+
+import unittest
+
+import lsst.utils.tests
+from lsst.daf.butler.instrument_tests import InstrumentTests, InstrumentTestData
+import lsst.obs.decam
+
+
+class TestDarkEnergyCamera(InstrumentTests, lsst.utils.tests.TestCase):
+    def setUp(self):
+        physical_filters = {"VR DECam c0007 6300.0 2600.0",
```

Comment:
Does this list of filters have to match the actual list of filters? If so, why can't we use the actual list? Are we testing that the two lists are the same? Or are we testing something else but we need filters?

Reply:
I wanted a way to confirm that the list of physical filter names that was registered with the gen3 butler was what the obs maintainer expected. I'm open for other ways of testing the registration system.

Reply:
It depends on what is being tested.

Reply:
Repeating a related comment in obs_base, because it's relevant here: I think it'd be good to spot-check that a few physical_filters were added explicitly by duplicating them, but expecting the entire list to be duplicated just seems repetitive and fragile.

Reply:
I've made the test a "subset" test, but left the list here and in obs_subaru as they are.